src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -35,6 +36,7 @@
             + "[" + PREFIX_CURRENT_WEIGHT + "CURRENT_WEIGHT] "
             + "[" + PREFIX_TARGET_WEIGHT + "TARGET_WEIGHT] "
             + "[" + PREFIX_HEIGHT + "HEIGHT]\n"
+            + "[" + PREFIX_REMARK + "REMARK]\n"
```

Comment:
There is an extra new line. Should be `+ "[" + PREFIX_HEIGHT + "HEIGHT] "`

Reply:
removed